lib/src/file_util.rs

```diff
@@ -334,9 +334,18 @@ mod platform {
     /// Developer Mode setting, which is stored in the registry key below.
     pub fn check_symlink_support() -> io::Result<bool> {
         let hklm = RegKey::predef(HKEY_LOCAL_MACHINE);
-        let sideloading =
-            hklm.open_subkey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock")?;
-        let developer_mode: u32 = sideloading.get_value("AllowDevelopmentWithoutDevLicense")?;
+        let sideloading = match hklm
+            .open_subkey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock")
+        {
+            Ok(key) => key,
+            Err(ref e) if e.kind() == io::ErrorKind::NotFound => return Ok(false),
+            Err(e) => return Err(e),
+        };
+        let developer_mode: u32 = match sideloading.get_value("AllowDevelopmentWithoutDevLicense") {
+            Ok(val) => val,
+            Err(ref e) if e.kind() == io::ErrorKind::NotFound => return Ok(false),
+            Err(e) => return Err(e),
+        };
```

Comment:
Can you update commit description to include why this change is needed. It's also good to split a commit that "fixes check_symlink_support".